### PR TITLE
feat: configurable view decorators

### PR DESCRIPTION
**Pull request by michalsn**
This PR adds the ability to disable decorators.

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Configuration
+
+To make changes to the config file, we have to have our copy in the `app/Config/MHtmx.php`. Luckily, this package comes with handy command that will make this easy.
```
Comment (datamweb) [inline docs/configuration.md]:
```suggestion
To make changes to the config file, we have to have our copy in the `app/Config/Htmx.php`. Luckily, this package comes with handy command that will make this easy.
```

File: docs/configuration.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- [$errorModalDecorator](#$errorModalDecorator)
+- [$skipViewDecoratorsString](#$skipViewDecoratorsString)
+
+### $toolbarDecorator
+
+This allows us to disable the `ToolbarDecorator` class. Please read [Debug Toolbar](debug_toolbar.md) page for more information.
+
+### $errorModalDecorator
+
+This allows us to disable thr `ErrorModalDecorator` class. Please read [Error handling](error_handling.md) page for more information.
```
Comment (datamweb) [inline docs/configuration.md]:
```suggestion
This allows us to disable the `ErrorModalDecorator` class. Please read [Error handling](error_handling.md) page for more information.
```

File: src/Commands/HtmxPublish.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            $publisher->addPaths([
+                'Config/Htmx.php',
+            ])->merge(false);
+        } catch (Throwable $e) {
+            $this->showError($e);
+
+            return;
+        }
+
+        foreach ($publisher->getPublished() as $file) {
```
Comment (datamweb) [inline src/Commands/HtmxPublish.php:35]: I think you should go for the new version. I know you are resistant to this.
However, version 4.4.2 has a security problem and I personally do not like it.

https://codeigniter4.github.io/userguide/libraries/publisher.html#replace-string-file-array-replaces-bool

**Review by datamweb — CHANGES_REQUESTED** [review]

File: src/Commands/HtmxPublish.php (hunk 3 above)
Comment (michalsn) [reply src/Commands/HtmxPublish.php:35]: Thanks, I wasn't aware there was a new method `replace()`, however, there is no much difference so I don't feel pressure for a change.
What security problem are you referring to?

Comment (datamweb) [reply src/Commands/HtmxPublish.php:35]: No big deal. Feel free to continue.
I meant,
https://github.com/codeigniter4/CodeIgniter4/security/advisories/GHSA-hwxf-qxj7-7rfj

Comment (michalsn) [reply src/Commands/HtmxPublish.php:35]: Well, this is a CLI command - used only by the code owner. Although the issue is true, I would say Commands are not affected.

**Comment by michalsn** [comment]
@datamweb Thank you for the code review.

**Pull request merged**